site/README.md

```diff
@@ -0,0 +1,18 @@
+# IPFS Distributions
+
+
+## Development
+
+Harpjs setup with
+
+* Live reload using browser-sync
+* JavaScript
+  * bundeling using webpack
+  * transpiling using babel
+* Linting using eslint
```

Comment:
There are a couple of major points why I think we need them

* Manage dependencies using npm
* Consistency with other browser based modules like `js-ipf-api`
* Make my life easier when working on the code (livereload is really a must for web dev for me)

The only thing that might not be needed is harpjs, though it was just a lot faster for me to use something working like that, than implementing all the tooling manually.

For me this is a lot simpler and more straight forward setup, especially while working with it than it was before. Also this enables me to add things like proper minification and asset bundeling which we need to have to ship this.